packages/charts/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -404,17 +405,21 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
   };
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  private _getStackedData = (keys: string[], dataSet: any) => {
+  private _getStackedData = (keys: string[], dataSet: any, mode: 'tozeroy' | 'tonexty' = 'tonexty') => {
```

Comment:
>private [](http://example.com/codeflow?start=2&length=7)

dont change existing logic. Add a new function if mode is tozeroy.

Reply:
that will require checking mode value every time the function is called.

Reply:
lets do that. I dont want any change in the existing logic right now.